assets/js/utils/rtcUtils.ts

```diff
@@ -27,34 +31,45 @@ export const mergedScreenRef: MergedScreenRef = {
 
 const sourceIds: { audio: string; video: string } = { audio: "", video: "" };
 
-export const findTrackByType = (client: Client, sourceType: SourceType) => {
-  return presenterStreams[client.email].getTracks().find((elem) => elem.kind == sourceType);
+export const findTrackByType = (client: Client | Presenter, sourceType: SourceType) => {
+  return presenterStreams[client.email]?.getTracks().find((elem) => elem.kind == sourceType);
```

Comment:
Why do we need `?` here? If removing it leads to some errors that's mean we call findTrackByType for client that doesn't exists so this is a bug.

Reply:
Yeah, that was right. The function `changeIsTrackEnabled` sometimes was called before the webrtc init. I've changed this function instead of adding this `?` sign.
Hope it is fine. 😅